pwndbg/commands/__init__.py

```diff
@@ -259,15 +263,15 @@ def split_args(self, argument):
 
 class ArgparsedCommand(object):
     """Adds documentation and offloads parsing for a Command via argparse"""
-    def __init__(self, parser_or_desc):
+    def __init__(self, parser_or_desc, aliases=[]):
```

Comment:
This could turn into *default argument trap* (i.e. the `[]` is created _only once_ and multiple calls to this method without passing aliases uses the the default value - the same object) but this could be easily spotted imho.

For reference: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments